src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -65,7 +66,16 @@ public class TwilioRestClient : ITwilioRestClient
             _username = username;
             _password = password;
 
-            AccountSid = accountSid ?? username;
+            _accountSid = accountSid;
+            //Validate prefix in accountSid, https://www.twilio.com/docs/glossary/what-is-a-sid#common-sid-prefixes
+            if (_accountSid?.StartsWith("AC", StringComparison.Ordinal) == false)
```

Comment:
Logic looks sound to me, but some tests would ease my mind more.

Reply:
Thanks, I will see what I can do about adding tests.